cmd/provider/main.go

```diff
@@ -289,58 +460,15 @@ func main() {
 	})
 	kingpin.FatalIfError(err, "Cannot create Kube client")
 
-	kingpin.FatalIfError(ctrl.NewWebhookManagedBy(mgr).
-		For(&providercephv1alpha1.Bucket{}).
-		WithValidator(bucket.NewBucketValidator(backendStore)).
-		Complete(), "Cannot setup bucket validating webhook")
+	setupBucketWebhook(mgr, backendStore)
+	setupProviderConfigControllers(mgr, o, backendStore, kubeClientUncached, log, *s3Timeout, *backendMonitorInterval, autoPauseBucket)
```

Comment:
For these funcs with a lot of parameters I'd rather keep them on new lines for readability. Same for others below.
```suggestion
	setupProviderConfigControllers(
	  mgr, 
	  o, 
	  backendStore,
	  ...
	)
```